specification/compute/resource-manager/Microsoft.Compute/stable/2021-11-01/compute.json

```diff
@@ -8113,6 +8113,60 @@
           "final-state-via": "azure-async-operation"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/hostGroups/{hostGroupName}/hosts/{hostName}/reboot": {
```

Comment:
The api should be called restart not reboot to be inline with the similar action for VirtualMachine resource.

Reply:
I'm fine with the rename, however this will delay the SDK PR quite a bit. In order to change the API name we'd have to merge this CRP PR ([that I created just now](https://msazure.visualstudio.com/One/_git/Compute-CPlat-Core/pullrequest/5408819)) to make changes to DedicatedHostsController.cs to perform the rename.

Then, wait for it to roll out before running a live test as part of the SDK PR, since the live test requires the new feature code to be in a live region.

Is there a simpler way to rename Reboot to Restart, or are you fine with the above?